### DISCUSSION TITLE
Kitty config file is not loading on startup (MacOS)

### DISCUSSION THREAD

**Discussion by c4m1l0-lending** [kovidgoyal/kitty#3774] [Q&A]
Hi! I have a trouble loading my `kitty.conf` on startup, lauching kitty from the icon in macOS Big Sur loads the default configuration, but when I enter `kitty` on the command line it loads my `kitty.conf` file. Could you please give me any suggestion to fix this?

**Comment by eatkins** [comment]
I've also run into this on Catalina and would appreciate some help.

**Comment by kovidgoyal** [comment]
Read the help for --config in kitty --help it will explain exactly how and from where kitty.conf is loaded. If you are getting different loads when launching from different places it will be because the environment variables in those different places are different.

**Reply by eatkins** [reply]
Yeah, it seems to be working ok now. I don't know what was messed up about my environment before but I was having an issue where when I ran kitty from the command line the configuration was loaded from ~/.config/kitty/kitty.conf but when I opened the application via a dock icon it didn't. Unfortunately I don't have a repro anymore so I'll just apologize for the noise. But since I'm here, I'll also take the opportunity to say that I am very impressed by kitty. Nice work.

**Comment by wongjiahau** [comment]
I have this issue as well on macOS Monterey (12.2). It seems like I have to log out and log in again from my macOS account for the `kitty.conf` to be appropriately reloaded (as in opening Kitty from Dock or Spotlight). 

I changed my default shell to `fish`, not sure if that affects anything.
